### PR TITLE
release: Ensure all release periodics have cluster in tag_specification

### PR DESCRIPTION
**Pull request by smarterclayton**

### CODE REVIEW

**Comment by smarterclayton** [comment]
@hongkailiu rehearse really blew this page up - we might want to cap or improve the behavior in this mode

File: ci-operator/jobs/openshift/release/openshift-release-release-4.5-periodics.yaml [hunk 1]
```diff
@@ -39,10 +39,14 @@ periodics:
         fi
 
         # verify the tests pass
-        ci-operator --artifact-dir=$(ARTIFACTS) --secret-dir=/usr/local/pull-secret \
-          --secret-dir=/usr/local/e2e-aws-cluster-profile --template=/usr/local/e2e-aws \
-          --input-hash=$(BUILD_ID) --input-hash=$(JOB_NAME) \
+        ci-operator \
+          --artifact-dir=$(ARTIFACTS) \
+          --kubeconfig=/etc/apici/kubeconfig \
           --lease-server=http://boskos \
```
Comment (stevekuznetsov) [inline ci-operator/jobs/openshift/release/openshift-release-release-4.5-periodics.yaml]: These need to be the route, not the svc

Comment (smarterclayton) [reply ci-operator/jobs/openshift/release/openshift-release-release-4.5-periodics.yaml]: ok

Comment (smarterclayton) [reply ci-operator/jobs/openshift/release/openshift-release-release-4.5-periodics.yaml]: fixed

**Pull request merged**